### PR TITLE
IssueID #1209 - (NOT FOR MERGING YET) Display Themed Guidance For Organisation on "Customis…

### PR DESCRIPTION
**Pull request by johnpinto1**
…ed Template" Preview.

Fix for issue #1209.

### CODE REVIEW

File: app/views/org_admin/questions/_show.html.erb [hunk 1]
```diff
@@ -58,6 +58,36 @@
           <% end %>
         </dl>
       </div>
+      <div class="col-md-4">
+        <% if guidance.present? %>
```
Comment (briri) [inline app/views/org_admin/questions/_show.html.erb]: This is referencing the "annotation" guidance associated with the question. Themed guidance should be independent of that.

File: app/views/org_admin/questions/_show.html.erb [hunk 2]
```diff
@@ -58,6 +58,36 @@
...
+          <% themes_q = question.themes %>
+          <% if !themes_q.nil? %>
+            <div class="accordion" id="<%= question.id %>-guidance">
+            <% themes_q.each do |theme| %>
+              <% theme.guidances.each do |guidance| %>
+                <% GuidanceGroup.where(org_id: current_user.org_id).each do |guidance_group| %>
```
Comment (briri) [inline app/views/org_admin/questions/_show.html.erb]: I would switch this up to get the guidance groups first and then use the guidance group ids to filter out the guidance you want on the DB side. (See code snippet in my final notes)

**Comment by johnpinto1** [comment]
@xsrust and @briri - In my last commit https://github.com/DMPRoadmap/roadmap/commit/e81227278c0bb85a4bfd28e1e98ea8ae7469ec49 I have made the changes @briri requested. However,  I am having difficulty changing the **+/-** icon for themed guidance links. I have a new javascript function for adding/removing the appropriate **fa-plus** or **fa-minus** class in  https://github.com/DMPRoadmap/roadmap/commit/e81227278c0bb85a4bfd28e1e98ea8ae7469ec49#diff-e4eeba6b7c41c1d0b489059c4e01fa2e  **app/javascript/utils/accordionFaIcons.js** which is used in https://github.com/DMPRoadmap/roadmap/commit/e81227278c0bb85a4bfd28e1e98ea8ae7469ec49#diff-34134a807d528ba7c62c417175f4e509 **app/javascript/views/org_admin/phases/new_edit.js** (which has js code for initialising the questions). But the code does not work. I am sure I am doing something silly here. 

**Screenshots of Themed Guidance Changes**

**Themed Guidance Collapsed:**
selection_117

**Themed Guidance Expanded:**
selection_118

**

**Comment by briri** [comment]
Hey @johnpinto1. I did a little investigation into this and I think you want to use bootstrap panels instead of accordion here. Take a look at this one which is used on the Write Plan page to display themed guidance on the right of each question. 
https://github.com/DMPRoadmap/roadmap/blob/4d9665ef8c76736ac750fe6ed0763da26be2a0f3/app/views/guidance_groups/_index_by_theme.html.erb#L17

I think if you update your divs and their attributes to follow that pattern then the +/- should automatically start working for you.

**Comment by johnpinto1** [comment]
Thanks @briri. Your suggestion gives the Theme Guidance a nicer look. The changing of +/- icons for the Theme Guidance is still an issue. I tried copying the code in **panelHeadings.js** to **app/javascript/views/org_admin/phases/new_edit.js** in tha Ajax success section with no success.

**Pull request merged**